skbio/stats/composition.py

```diff
@@ -0,0 +1,375 @@
+r"""
+Composition Statistics (:mod:`skbio.stats.composition`)
+=======================================================
+
+.. currentmodule:: skbio.stats.composition
+
+This module provides functions for compositional data analysis.
+
+Many 'omics datasets are inheriently compositional - meaning that they are best
+interpreted as proportions or percentages rather than absolute counts.
+
+Formally, :math:`x` is a composition if :math:`\sum_{i=0}^D x_{i} = c` and
+:math:`x_{i} > 0`, :math:`1 \leq i \leq D`  and :math:`c` is a real valued
```

Comment:
Is $x_i > 0$ really a requirement? Not allowing null abundances seems restrictive.